CMSgov/design-system
# [WNMGDS-2835] Adds figma links to docs site

## Summary
- JIRA link: https://jira.cms.gov/browse/WNMGDS-2835
- Removes links to sketch assets on the docs site and replaces those with links to figma pages based on whichever theme is currently in use

## How to test

1. This branch can either be pulled down locally or viewed at https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/
2. Go through each updated docs site page using each theme (cmsgov, core, healthcare, medicare) and click the figma link and verify that the figma page corresponds with the docs site page content

- [x] [accordion](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/accordion)
- [x] [accordion](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/accordion)
- [x] [alert](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/alert)
- [x] [autocomplete](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/autocomplete)
- [x] [badge](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/badge)
- [x] [button](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/button)
- [x] [card](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/card)
- [x] [checkbox](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/checkbox)
- [x] [date-picker](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/date-field/date-picker)
- [x] [multi-input-date-field](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/date-field/multi-input-date-field)
- [x] [single-input-date-field](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/date-field/single-input-date-field)
- [x] [drawer](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/drawer)
- [x] [dropdown](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/dropdown)
- [x] [filter-chip](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/filter-chip)
- [x] [healthcare-footer](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/footer/healthcare-footer)
- [x] [medicare-footer](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/footer/medicare-footer)
- [x] [healthcare-header](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/header/healthcare-header)
- [x] [medicare-header](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/header/medicare-header)
- [x] [icon](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/icon)
- [x] [healthcare-logo](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/logos/healthcare-logo)
- [x] [hhs-logo](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/logos/hhs-logo)
- [x] [medicare-logo](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/logos/medicare-logo)
- [x] [modal-dialog](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/modal-dialog)
- [x] [month-picker](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/month-picker)
- [x] [pagination](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/pagination)
- [x] [radio](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/radio)
- [x] [review](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/review)
- [x] [skip-nav](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/skip-nav)
- [x] [spinner](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/spinner)
- [ ] [stars](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/stars)
- [x] [step-list](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/step-list)
- [x] [table](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/table)
- [x] [tabs](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/tabs)
- [x] [text-field/text-field](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/text-field/text-field)
- [x] [tooltip](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/tooltip)
- [x] [usa-banner](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/usa-banner)
- [x] [vertical-navigation](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/components/vertical-navigation)
- [ ] [layout-grid](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/foundation/layout-grid/layout-grid)
- [x] [spacing](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/foundation/spacing)
- [x] [body](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/foundation/typography/body)
- [x] [headings](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/foundation/typography/headings)
- [x] [links](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/foundation/typography/links)
- [x] [lists](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/foundation/typography/lists)
- [ ] [one-col-page-layout](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/layouts/Healthcare/one-col-page-layout)
- [x] [documentation-page](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/layouts/documentation-page)
- [x] [error-validation](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/patterns/Forms/error-validation)
- [x] [measure](https://cmsgov.github.io/design-system/branch/kniedermaier/WNMGDS-2835-update-docs-site/utilities/text/measure)

## Checklist
